### PR TITLE
docs(showcase): add @vedantmgoyal2009 docs (draft/do-not-merge)

### PR DESCRIPTION
**Pull request by vedantmgoyal2009**
## Motivation

Add my website to showcase.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes.

### CODE REVIEW

**Comment by slorber** [comment]
we need the screenshot

Also your site is a newly initialized default template: please submit it once it is a real doc site with real content

**Comment by vedantmgoyal2009** [comment]
Hello again, I have pushed the documentation now. Can you please re-open the the PR?

**Comment by vedantmgoyal2009** [comment]
cc @Josh-Cena @lex111 @slorber

**Comment by slorber** [comment]
@vedantmgoyal2009 the screenshot it still missing in this PR

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1980,6 +1980,14 @@ const Users: User[] = [
     source: 'https://github.com/HiChen404/MyWikiSite',
     tags: ['opensource', 'personal'],
   },
+  {
+    title: 'Vedant\'s Projects\' Docs',
+    description: 'Documentation of all projects by @vedantmgoyal2009',
+    preview: require('./showcase/vedantmgoyal2009.png'),
```
Comment (slorber) [inline website/src/data/users.tsx:1986]: need to upload this source

**Pull request closed without merging**